static/app/components/events/interfaces/crashContent/exception/utils.tsx

```diff
@@ -20,6 +28,46 @@ export function isFrameFilenamePathlike(frame: Frame): boolean {
   );
 }
 
+interface RenderLinksInTextProps {
+  exceptionText: string;
+}
+
+export const renderLinksInText = ({
+  exceptionText,
+}: RenderLinksInTextProps): ReactElement => {
+  if (!exceptionText) {
+    return <Fragment />;
+  }
+
+  const urlRegex =
+    /https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9]{1,6}\b(?:[-a-zA-Z0-9@:%_\+.~#?&\/=,\[\]]*)/gi;
+  const parts = exceptionText.split(urlRegex);
+  const urls = exceptionText.match(urlRegex) || [];
+
+  const elements = parts.flatMap((part, index) => {
+    const isUrlValid = urls[index] && isUrl(urls[index]);
+    const link = isUrlValid ? (
+      <Link
```

Comment:
This should now be a `Button` element because it isn't actually linking anywhere, it just opens a modal. To make it look like a link you should be able to do `<Button priority="link" />`

Reply:
After discussing, we actually can't style a button to look the way we want it to (because button's are always `inline-block` but we need `inline`).

We should render a link with the correct href, but ensure that the behavior is overridden and a modal is shown instead:

```tsx
<ExternalLink href={url} onClick={e => e.preventDefault(); openModal()}>
  {url}
</ExternalLink>
```